app/models/landing_page/compound_block.rb

```diff
@@ -0,0 +1,20 @@
+class LandingPage::CompoundBlock < LandingPage::BaseBlock
```

Comment:
One of those rare cases that might save a few future head-scratches with a short comment now:

```suggestion
# Compound blocks are blocks that are made up of several child blocks
# e.g. a 'hero' block contains both an image block and a title overlay block

class LandingPage::CompoundBlock < LandingPage::BaseBlock
```

^ I don't actually know what blocks are involved within the Hero block, as the associated test just specifies `"blocks" => [{ "type" => "some-block-type" }]`. So a comment might help clarify.

Also if I've understood its purpose properly, `CompositeBlock` might be a better name for it? 🤔

Reply:
Done in 38c9c2a